src/gravel/controllers/services.py

```diff
@@ -332,6 +340,28 @@ def get_pool(name: str) -> CephOSDPoolEntryModel:
             # do nothing else, the service still works without an authorized
             # client.
 
+    def _create_nfs(self, svc: ServiceModel) -> None:
+        # create a cephfs
+        self._create_cephfs(svc)
+
+        # create an generic NFS service
+        nfs_svc_id = 'gravel'
+        try:
+            NFSService().create(nfs_svc_id, placement='*')
```

Comment:
does this mean we only get to have one NFS service?

Reply:
yes, one NFS service an NFS export per cephfs.

co-location of NFS services is wip: https://github.com/ceph/ceph/pull/40543

Reply:
Okay, so, I understand this might seem intuitive but I had to go run the commands myself to ensure the creation command is in fact idempotent. Mind adding a note there stating that this will indeed call `nfs cluster create` multiple times, once per service creation, but that the underlying command is idempotent as thus we get away with it?

I know this feels like a little thing, but from the code, as is, really seems like we could just be bailing out through an exception should the service already exist.

Reply:
actually, good catch! I think your intuition is right .. 

The command appears to be idempotent, but maybe it should not be, because if the supplied placement spec happens to differ, this is not currently updated via the `create` command. Likewise, there is also no guarantee the current behavior won't change out from under us later.

I've fixed up the commit to first look for an existing cluster and update the placement or create a new cluster if one does not exist.